src/app/App.tsx

```diff
@@ -137,13 +139,27 @@ const App: FC = (): JSX.Element => {
   return (
     <ThemeProvider theme={themeController[state.settings.theme]!}>
       <AppWrapper>
+        <Helmet>
+          <meta charSet='utf-8' />
+          <meta
+            name='description'
+            content='Weather app to check current weather in any location'
+          />
+
+          <title>Weather App</title>
```

Comment:
FYI. This works but shows the change between the original title (React App) and the new title. Not very important but good to know for later